src/app/code/community/Wf/Bananastand/Helper/Data.php

```diff
@@ -143,5 +143,85 @@ public function isDebugMode()
         return Mage::getStoreConfigFlag('banana/general/debug_mode');
     }
 
+    /**
+     * @return json - The contents of the cart as a json string.
+     */
+    public function getCartJson()
+    {
+        $cartItems = Mage::getModel('checkout/cart')->getItems();
+        if (empty($cartItems)) {
+          return "[]";
+        }
+        return json_encode($cartItems->getData());
+    }
+
+    /**
+     * Format a mage order object into JSON for sending to VIA js api
+     * @param Mage_Sales_Model_Order $order
+     * @return JSON
+     */
+    public function jsonifyOrder($order)
+    {
+        $items = array();
+
+        foreach ($order->getAllItems() as $item) {
+            $product = $item->getProduct();
+            $items[] = array(
+                'name'     => $item->getName(),
+                'total'    => $item->getPrice(),
+                'quantity' => $item->getQtyOrdered(),
+                'product'  => array(
+                    'id'            => $product->getId(),
+                    'name'          => $product->getName(),
+                    'status'        => $product->getStatus(),
+                    'in_stock'      => $product->isInStock(),
+                    'url'           => $product->getProductUrl(),
+                    'thumbnail_url' => $product->getThumbnailUrl(),
+                    'platform_data' => $product->getData()
+                ),
+            );
+        }
 
+        $customer_id = $order->getCustomerId();
+        $customer;
+        if (!empty($customer_id)) {
+            $customerData = Mage::getModel('customer/customer')->load($customer_id);
+            $addressObj = $customerData->getPrimaryShippingAddress();
+            $address;
+            if (!empty($addressObj)) {
+                $address = $addressObj->getData();
+            }
+            $customer = array(
+                'id'            => $customer_id,
+                'first_name'    => $customerData->getFirstname(),
+                'last_name'     => $customerData->getLastname(),
+                'email'         => $customerData->getEmail(),
+                'address'       => $address
+            );
+        }
+
+        $currency_code = Mage::app()->getStore()->getCurrentCurrencyCode();
```

Comment:
Can you get the currency code from the order instead? I hope so, but if not then we can try something else